fallzahlen_kanton_total_csv/COVID19_Fallzahlen_Kanton_BS_total.csv

```diff
@@ -1,35 +1,35 @@
+2020-03-16,12:00,BS,"",144,"","","","",4,https://www.coronavirus.bs.ch/nm/2020-tagesbulletin-coronavirus-144-bestaetigte-faelle-im-kanton-basel-stadt-gd.html,126,""
+2020-03-17,12:00,BS,"",165,30,"","",25,4,https://www.coronavirus.bs.ch/nm/2020-tagesbulletin-coronavirus-165-bestaetigte-faelle-im-kanton-basel-stadt-gd.html,145,""
+2020-03-18,11:15,BS,"",182,40,"","",36,4,https://www.coronavirus.bs.ch/nm/2020-tagesbulletin-coronavirus-182-bestaetigte-faelle-im-kanton-basel-stadt-treffen-mit-allen-spitaelern-gd.html,164,""
+2020-03-19,11:00,BS,"",222,40,"","",44,4,https://www.coronavirus.bs.ch/nm/2020-tagesbulletin-coronavirus-222-bestaetigte-faelle-basel-stadt-trifft-gemeinsam-mit-den-spitaelern-vorkehrungen-fuer-intensiv--und-beatmungskapazitaeten-gd.html,199,""
+2020-03-20,10:00,BS,"",272,45,"","",46,4,https://www.coronavirus.bs.ch/nm/2020-tagesbulletin-coronavirus-272-bestaetigte-faelle-im-kanton-basel-stadt-gd.html,232,""
+2020-03-21,10:00,BS,"",299,46,"","",57,5,https://www.coronavirus.bs.ch/nm/2020-tagesbulletin-coronavirus-299-bestaetigte-faelle-im-kanton-basel-stadt-gd.html,242,""
+2020-03-22,10:30,BS,"",358,50,"","",73,5,https://www.coronavirus.bs.ch/nm/2020-tagesbulletin-coronavirus-358-bestaetigte-faelle-im-kanton-basel-stadt-gd.html,265,""
+2020-03-23,10:00,BS,"",376,56,"","",78,5,https://www.coronavirus.bs.ch/nm/2020-tagesbulletin-coronavirus-376-bestaetigte-faelle-im-kanton-basel-stadt-gd.html,276,""
+2020-03-24,09:00,BS,"",414,58,"","",105,5,https://www.coronavirus.bs.ch/nm/2020-tagesbulletin-coronavirus-414-bestaetigte-faelle-im-kanton-basel-stadt-gd.html,291,""
+2020-03-25,10:00,BS,"",466,58,"","",128,8,https://www.coronavirus.bs.ch/nm/2020-tagesbulletin-coronavirus-466-bestaetigte-faelle-im-kanton-basel-stadt-gd.html,307,""
+2020-03-26,10:30,BS,"",505,74,8,"",155,12,https://www.coronavirus.bs.ch/nm/2020-tagesbulletin-coronavirus-505-bestaetigte-faelle-im-kanton-basel-stadt-gd.html,326,20
+2020-03-27,10:00,BS,"",534,76,8,"",191,13,https://www.coronavirus.bs.ch/nm/2020-tagesbulletin-coronavirus-534-bestaetigte-faelle-im-kanton-basel-stadt-gd.html,343,19
+2020-03-28,10:00,BS,"",573,79,11,"",211,13,https://www.coronavirus.bs.ch/nm/2020-tagesbulletin-coronavirus-573-bestaetigte-faelle-im-kanton-basel-stadt-gd.html,365,20
+2020-03-29,10:00,BS,"",609,87,12,"",228,15,https://www.coronavirus.bs.ch/nm/2020-tagesbulletin-coronavirus-609-bestaetigte-faelle-im-kanton-basel-stadt-gd.html,373,22
+2020-03-30,10:00,BS,"",621,90,12,"",263,15,https://www.coronavirus.bs.ch/nm/2020-tagesbulletin-coronavirus-621-bestaetigte-faelle-im-kanton-basel-stadt-gd.html,376,20
+2020-03-31,10:00,BS,"",628,105,15,"",292,16,"https://www.coronavirus.bs.ch/nm/2020-tagesbulletin-coronavirus-628-bestaetigte-faelle-im-kanton-basel-stadt-gd.html https://www.coronavirus.bs.ch/nm/2020-tagesbulletin-coronavirus-691-bestaetigte-faelle-im-kanton-basel-stadt-gd.html",381,23
```

Comment:
Was this itentional to update the number of confirmed cases here? Before it was 657.

Reply:
@metaodi Oops sorry, that was a merging error, thanks for spotting it! Now it's fixed. 657 is correct (I had to correct yesterday's number based on today's press release).